### DISCUSSION TITLE
Example programs for learning

### DISCUSSION THREAD

**Discussion by eduardsik** [visose/Robots#207] [Q&A] [answered]
Are there any sample programs available for the plugin? Apart from those in it.

I found this, in case it's useful for others: 
https://ms-kb.msd.unimelb.edu.au/msd-robotics-lab/software/rhino-and-grasshopper-1/create-a-basic-robot-program

**Comment by visose** [comment, marked as answer]
You can also find video tutorials on YouTube, although they are created by third parties.
Some examples:
https://www.youtube.com/watch?v=vAe47zN-d48&list=PLqtxhH1qb3Mw5A_YbvHDfrq4DNNfLtcW-
https://www.youtube.com/watch?v=HV44DDlDqmQ
https://www.youtube.com/watch?v=-JtDSXmi1YI
